ppr-ui/src/store/actions/actions-model.ts

```diff
@@ -169,7 +169,10 @@ export const setSearchDebtorName: ActionIF = ({ commit }, debtorName: Individual
 export const setSearchHistory: ActionIF = ({ commit }, searchHistory: Array<SearchResponseIF>): void => {
   // need to set .loadingPDF so that the loader circle triggers when set
   //  - if it starts as undefined it wont trigger on change
-  for (let i = 0; i < searchHistory?.length || 0; i++) { searchHistory[i].loadingPDF = false }
+  for (let i = 0; i < searchHistory?.length || 0; i++) {
+    searchHistory[i].loadingPDF = false
+    searchHistory[i].isPdfRequested = false
+  }
```

Comment:
Had to revert to original code, as the following was failing the tests with some strange errors 🤷🏻 :
 ```
 (searchHistory || []).forEach(item => {
    console.log(item)
    item.loadingPDF = false
    item.isPdfRequested = false
  })
```
@cameron-eyds